sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/csv/CSVSuite.scala

```diff
@@ -447,7 +446,7 @@ class CSVSuite extends QueryTest with SharedSQLContext with SQLTestUtils {
 
     verifyCars(cars, withHeader = true, checkValues = false)
     val results = cars.collect()
-    assert(results(0).toSeq === Array(2012, "Tesla", "S", "null", "null"))
+    assert(results(0).toSeq === Array(2012, "Tesla", "S", null, null))
```

Comment:
This is being tested against the data as below:

```
year,make,model,comment,blank
"2012","Tesla","S",null,

1997,Ford,E350,"Go get one now they are going fast",
null,Chevy,Volt

```

Since the header is `year`,`make`,`model`,`comment`,`blank`, this should produce the values `2012`,`Tesla`,`S`,`null`,`null`.